src/pages/Home/components/Days/Card.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Day } from './Days'
+import {GlobalSvgSelector} from "../../../../assets/icons/global/GlobalSvgSelector";
+import {CardStyle, DayInfo, DayStyle, Img, DayTemp, NightTemp, Info} from "./DaysStyles";
+interface Props{
+  day: Day;
+}
+
+export const Card =({day}:Props)=>{
```

Comment:
Better call it DayCardComponent
```suggestion
export const DayCardComponent = ({ day }: Props) => {
```